lib/api/apiUtils/object/coldStorage.js

```diff
@@ -244,14 +244,22 @@ function _updateRestoreInfo(objectMD, restoreParam, log) {
  *
  */
 function startRestore(objectMD, restoreParam, log, cb) {
-    log.info('Validating if restore can be done or not.');
+    log.debug('Validating if restore can be done or not.');
     const checkResultError = _validateStartRestore(objectMD, log);
     if (checkResultError) {
+        log.debug('Restore cannot be done.', {
+            error: checkResultError,
+            method: 'startRestore'
+        });
         return cb(checkResultError);
     }
-    log.info('Updating restore information.');
+    log.debug('Updating restore information.');
     const updateResultError = _updateRestoreInfo(objectMD, restoreParam, log);
     if (updateResultError) {
+        log.debug('Failed to update restore information.', {
+            error: updateResultError,
+            method: 'startRestore'
+        });
         return cb(updateResultError);
     }
```

Comment:
should we dedup logs?
these are synchronous calls, not sure what it brings...

```suggestion
    const checkResultError = _validateStartRestore(objectMD, log);
    if (checkResultError) {
        log.debug('Restore cannot be done.', {
            error: checkResultError,
            method: 'startRestore'
        });
        return cb(checkResultError);
    }
    const updateResultError = _updateRestoreInfo(objectMD, restoreParam, log);
    if (updateResultError) {
        log.debug('Failed to update restore information.', {
            error: updateResultError,
            method: 'startRestore'
        });
        return cb(updateResultError);
    }
    log.debug('Validated and updated restore information');
```